src/Illuminate/Support/Collection.php

```diff
@@ -939,6 +960,25 @@ public function transform(callable $callback)
     }
 
     /**
+     * Transpose each item in the collection, interchanging the row and column indexes.
+     *
+     * @return static
+     */
+    public function transpose()
+    {
+        $values = $this->values();
+
+        if (! $values->forAll('is_array')) {
+            throw new InvalidArgumentException('Can only transpose multi-dimensional arrays.');
+        }
+
+        $values = $values->toArray();
```

Comment:
Semantically you probably want `all()` here instead of `toArray()`. `toArray()` calls `toArray()` on every element in the collection, which can have unexpected results if you have Eloquent models in there and stuff. In this particular case I don't think it actually changes the behavior because of the `is_array` check that's run first, but regardless `all()` is the right way to just grab the internal array from the collection.